nuxt.config.ts

```diff
@@ -1,8 +1,18 @@
 // https://nuxt.com/docs/api/configuration/nuxt-config
 export default defineNuxtConfig({
   ssr: false,
-  modules: ['@pinia/nuxt', '@vueuse/nuxt', '@nuxt/ui', 'nuxt-icon'],
+  modules: ['@pinia/nuxt', '@vueuse/nuxt', '@nuxt/ui', 'nuxt-icon', 'nuxt-open-fetch'],
   experimental: { typedPages: true },
   devtools: { enabled: true },
   ui: { global: true, icons: 'all' },
+  openFetch: {
+    disablePlugin: true,
+    clients: {
+      httpd: {
+        schema:
+          'https://seed.rhizoma.dev/raw/rad:z3yQUb9HDAC7TQrUDGkQsXDsYFj9G/b12552cf6803ecb9ddab58714b571684465dfe6d/api/radicle-httpd.yaml',
+        baseURL: 'https://seed.radicle.xyz/api/v1',
```

Comment:
Should be `http://0.0.0.0:8080` since that's the default of httpd 

![image](https://github.com/cytechmobile/radicle-planning-boards/assets/7542054/029d2d41-4df6-450a-bc11-131e1640f992)

If we want to hard-override that value in `plugins/httpd-client.ts` with a fixed string (for this PR, to be made dynamic later) that's fine by me. But keep in mind that radicle is a local app (p2p) and our client should work locally by default too.

Reply:
Updated it to `http://127.0.0.1:8080`, elected not to add the override in plugins for now, and add the dynamic one once needed.